src/content/ja/2019/caching.md

```diff
@@ -2,7 +2,7 @@
 part_number: IV
 chapter_number: 16
 title: キャッシング
-description: 2019Web年鑑のキャッシュの章は、キャッシュコントロール、有効期限、TTL、有効性、変化、Cookieの設定、アプリケーションキャッシュ、Service Worker、および機会について説明します。
+description: 2019Web Almanacのキャッシュの章は、キャッシュコントロール、有効期限、TTL、有効性、変化、Cookieの設定、アプリケーションキャッシュ、Service Worker、および機会について説明します。
```

Comment:
Should there be a space here? Or not required?
```suggestion
description: 2019 Web Almanacのキャッシュの章は、キャッシュコントロール、有効期限、TTL、有効性、変化、Cookieの設定、アプリケーションキャッシュ、Service Worker、および機会について説明します。
```

Reply:
Thanks, I fixed.